server/meshmodel/istio-base/1.16.0/v1.0.0/relationships/hierarchical_inventory_WASMFilterToEnvoyFilter.json

```diff
@@ -0,0 +1,71 @@
+{
+    "schemaVersion": "core.meshery.io/v1alpha2",
+    "version": "v1.0.0",
+    "kind": "Hierarchical",
+    "metadata": {
+      "description": "A hierarchical inventory relationship in which the configuration of (parent) component is patched with the configuration of other (child) component. Eg: The configuration of the EnvoyFilter (parent) component is patched with the configuration as received from WASMFilter (child) component."
+    },
+    "model": {
+      "schemaVersion": "core.meshery.io/v1beta1",
+      "version": "v1.0.0",
+      "name": "istio-base",
+      "model": {
+        "version": "v1.16.0"
+      },
+      "displayName": "Istio",
+      "category": {
```

Comment:
Soon we will eliminate this duplication of information. Who will simply be a ref to the model.